packages/flutter_tts/tizen/src/text_to_speech.cc

```diff
@@ -30,15 +30,27 @@ void OnError(tts_h tts, int utt_id, tts_error_e reason, void *user_data) {
 
 }  // namespace
```

Comment:
You might just inline the callbacks and move the code to `RegisterTtsCallback` for simplicity. Please also remove unnecessary casts `(void *)` in `RegisterTtsCallback` and `GetSupportedLanaguages`.

Reply:
done